### PR TITLE
Remove dynamic keyboard event mode

### PR DESCRIPTION
**Pull request by brichet**
This PR is a follow up of https://github.com/jupyterlab/lumino/pull/635, to remove an experimental implementation.

The keyboard event mode cannot be dynamically set up anymore, only when the application starts.

### CODE REVIEW

File: packages/application/src/index.ts [hunk 1]
```diff
@@ -623,7 +609,7 @@ export class Application<T extends Widget = Widget> {
    */
   protected addEventListeners(): void {
     if (this._bubblingKeydown) {
-      console.log('The keydown events are handled during bubbling phase');
+      console.debug('The keydown events are handled during bubbling phase');
```
Comment (fcollonval) [inline packages/application/src/index.ts:612]: do we want to keep this log?

Comment (brichet) [reply packages/application/src/index.ts:612]: Probably not

**Review by krassowski — APPROVED** [review]
Thanks!

**Review by fcollonval — APPROVED** [review]
Thanks

**Pull request merged**